### PR TITLE
Mozilla FO resolution

### PR DESCRIPTION
**Pull request by martinthomson**
I've taken some liberties here with respect to people.  Otherwise, this is a straight implementation of the recommendations in [Mozilla's formal objection](https://lists.w3.org/Archives/Public/public-new-work/2022Sep/0006.html) to our initial chartering attempt.

@seanturner, @AramZS, @samuelweiler, and @wseltzer probably need to pay attention.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -128,15 +128,22 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
               Chairs
             </th>
             <td>
-              <i class="todo">[chair name] (affiliation)</i>
+              <ul>
+                <li>Aram Zucker-Scharff (Washington Post)</li>
+                <li>Sean Turner (Sn3rd; Invited Expert)</li>
```
Comment (seanturner) [inline charter.html:133]: Minor nit.
```suggestion
                <li>Sean Turner (sn3rd; Invited Expert)</li>
```

File: charter.html [hunk 2]
```diff
@@ -128,15 +128,22 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
...
+         </tr>
           <tr>
             <th>
               Team Contacts
             </th>
             <td>
-              <span class="todo"><a href="mailto:">[team contact name]</a></span> <i class="todo">(0.1 <abbr title="Full-Time Equivalent">FTE</abbr>)</i>
+              <ul>
+                <li>Sam Weiler</li>
+                <li>Wendy Seltzer</li>
```
Comment (hober) [inline charter.html:144]: Wendy is no longer with the W3C, so I don't think it'd be appropriate to list her as a Team contact.

Comment (martinthomson) [reply charter.html:144]: Excellent point.  This wasn't true when I opened the PR, but here we are.

```suggestion
```

File: charter.html [hunk 3]
```diff
@@ -128,15 +128,21 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
...
-          </tr>
+         </tr>
           <tr>
             <th>
               Team Contacts
             </th>
             <td>
-              <span class="todo"><a href="mailto:">[team contact name]</a></span> <i class="todo">(0.1 <abbr title="Full-Time Equivalent">FTE</abbr>)</i>
+              <ul>
+                <li>Sam Weiler</li>
```
Comment (martinthomson) [inline charter.html:143]:
```suggestion
                <li>TBD (to be filled by W3C)</li>
```

**Pull request merged**